### PR TITLE
virttest: Add a new param discard granularity for a disk

### PR DESCRIPTION
**Pull request by peixiu**
Add a new param discard_granularity for a virtio disk

id: 2137732
Signed-off-by: Peixiu Hou <phou@redhat.com>

### CODE REVIEW

**Comment by peixiu** [comment]
Hi @YongxueHong @qingwangrh,

Could you help to review this patch? I added a new param discard_granularity.

Thanks~
Peixiu

File: virttest/qemu_devices/qdevices.py [hunk 1]
```diff
@@ -1157,9 +1157,11 @@ def _cmdline_json(self):
             # max_sectors from device ("driver": "virtio-scsi-pci")
             # num_queues from device ("driver": "virtio-scsi-pci")
             # virtqueue_size from device ("driver": "virtio-scsi-pci")
+            # discard_granularity from device ("driver": "scsi-hd",
+            # "driver": "virtio-blk-pci")
```
Comment (YongxueHong) [inline virttest/qemu_devices/qdevices.py:1161]:
```suggestion
            # discard_granularity from device ("driver": "scsi-hd") and
            # ("driver": "virtio-blk-pci")
```

Comment (peixiu) [reply virttest/qemu_devices/qdevices.py]: Thanks, updated~

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**